OneCommunityGlobal/HGNRest
# Mohammad - Middle name search on the Project team members search bar(Backend)

Add middle name searching to the project member search bar:
Fixes # (bug list priority medium)
Modify getUserBySingleName and getUserByFullName in userProfileController.js located in HGNRest. To allow for middle name querying.
Modify findUserProfiles in projectMembers.js in HighestGoodNetworkApp to extract and send the middle name to the backend.
Note: Users are stored with a firstName field and lastName field. The middle name should be concatenated to the end of the first name. The below table shows an example of how a user with a middle name is stored.

## Related PRS (if any):
To test this backend PR you need to checkout the #[2022](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2022) frontend PR.

## How to test:
check into current branch and check into frontend branch separately
do npm run build and npm start to run this PR locally
